src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -297,7 +297,6 @@ static String createQuery(Map<String, Object> params)
   private static List<Parameter> flattenParamsArray(Object[] params, String keyPrefix)
       throws InvalidRequestException {
```

Comment:
We can fix this in a different PR, but just wanted to flag that having both `flattenParamsArray` and `flattenParamsList` is pretty silly — these should be merged into one method that just takes an `Iterator`.

Reply:
Sigh agreed.